packages/mobile/src/screens/favorites-screen/useCollectionsScreenData.ts

```diff
@@ -117,38 +117,34 @@ export const useCollectionsScreenData = ({
         return []
       }
 
-      return fetchedCollectionIds.filter((collectionId) => {
+      const offlineCollectionsStatus = getOfflineCollectionsStatus(state)
+      const offlineCollectionIds = Object.keys(offlineCollectionsStatus).filter(
+        (k) => offlineCollectionsStatus[k] !== OfflineDownloadStatus.INACTIVE
```

Comment:
Can there be error state?

Reply:
Just checked Spotify and they show albums that are not fully downloaded so maybe this is okay?

Reply:
True, though we still only want to show them if we have the full collection metadata (name, cover art, owner, etc.)
Collection download success is only an indicator of the collection metadata being downloaded.

This is separate from the status we display on the indicator icon which is an aggregate of the collection's download status and the download status of each track in its contents.

Which is to say we should display collections if:
```
collection_download_status === SUCCESS &&
(downloaded_collection_tracks.length > 0 || collection_tracks.length === 0)
```

I believe the latter half is the logic we have below, and the only change we need is to use `=== SUCCESS` instead of `!== INACTIVE`